plugins/sqlfluff-templater-dbt/test/fixtures/dbt/profiles_yml/profiles.yml

```diff
@@ -4,10 +4,10 @@ default:
     dev:
       type: postgres
       host: localhost
```

Comment:
@jpy-git: Note for later: In the Docker Compose environment, this will need to be `postgres` instead. We should automate this later, but let's not worry about it right now, with two separate PRs in flight (dbt fix, Docker Compose). 🤯